src/pages/ListCandidatesVacancies.jsx

```diff
@@ -28,7 +63,7 @@ export const ListCandidatesVacancies = () => {
                 className="wrapper"
             >
                 <div>
-                    <OfferDetails />
+                    <OfferDetails {...JobCardData} />

```

Comment:
The problem of using `{...Data}` is that you can't know what exactly you pass to the component 🤔 . It's okay, but just think about it

Reply:
This is not validated by the PropTypes within the component?